projects/faq.md

```diff
@@ -0,0 +1,19 @@
+Q1: Why set up `projects/` folder?
```

Comment:
I think it will be more readable if we highlight the questions like making each question a section title.